core/word_eval.py

```diff
@@ -552,7 +552,16 @@ def _EvalBracedVarSub(self, part, part_vals, quoted):
 
     elif part.suffix_op:
       op = part.suffix_op
-      if op.tag == suffix_op_e.StringUnary:
+      if op.tag == suffix_op_e.StringNullary:
+        if op.op_id == Id.VOp0_P:
+          # TODO: Use dependency injection
```

Comment:
what's this comment?

Reply:
This is to possibly replace the global ui.PROMPT with dependency injection.  We can leave it in for this PR, but I probably want to refactor it later to follow the prevailing style.